connectors/src/connectors/intercom/temporal/sync_help_center.ts

```diff
@@ -228,7 +228,7 @@ export async function upsertCollectionWithChildren({
     folderId: internalCollectionId,
     title: collection.name,
     parents: collectionParents,
-    parentId: collectionParents.length > 2 ? collectionParents[1] : null,
+    parentId: collectionParents[1],
```

Comment:
as the type now shows, we at least have [`collectionInternalId`, ...maybe some parent collections, `helpCenterInternalId`], which is why this is correct